util/pyclient/bft_msgs.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, message):
 # Little Endian format with no padding
 # We don't include the msg type here, since we have to read it first to
 # understand what message is incoming.
-REQUEST_HEADER_FMT = "<HBQL"
+REQUEST_HEADER_FMT = "<HBQLL" # + str(CID_SIZE + 1) + 's'
```

Comment:
I don't understand this comment. We are only describing the header here. And the CID comes after the message anyway, so this seems to be incorrect. I suggest just removing it. It's clear from the code changes what's going on.

Reply:
Fixed